packages/frontend/src/layout/sideColumn/postsWidget.tsx

```diff
@@ -55,17 +53,24 @@ const RankingBlock = observer(({ post, ranking, hasUnderline }: Props) => {
 const PostsWidget = () => {
     const postContext = useContext(PostContext)
 
+    const sortByBoost = (a: Post, b: Post) => {
+        if (a.upvote >= b.upvote) return -1
+        else return 1
+    }
```

Comment:
don't know if we should sort the post ranking by `upvote - downvote`? @ChialiTsai @vimwitch